.circleci/test-deploy.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+orbs:
+  orb: 'cyber4all/orb@dev:<<pipeline.git.revision>>'
+  orb-tools: circleci/orb-tools@11.1
+
+jobs:
+  command-tests:
+    docker:
+      - image: 'cimg/base:current'
+    steps:
+      - checkout
+      - orb/check-docker-semver
+      - orb/setup-aws
+      - orb/setup-docker
+      - orb/tag-repo
+      - orb/validate-num-params
+      - orb/validate-str-params
+
+workflows:
+  test-deploy:
```

Comment:
Will test deploy actually deploy to eb/ecs/etc?

Reply:
No